components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -182,6 +193,11 @@ void BravePrefProvider::MigrateShieldsSettings(bool incognito) {
 
   // Finally clean this up now that Shields' settings have been migrated.
   prefs_->ClearPref("brave.migrate.content_settings.exceptions.plugins");
+
+#if defined(OS_ANDROID)
```

Comment:
You can clear this pref in the copy method

Reply:
Moved to `CopyPluginSettingsForMigration`